src/main/java/com/github/manolo8/darkbot/utils/StartupParams.java

```diff
@@ -0,0 +1,123 @@
+package com.github.manolo8.darkbot.utils;
+
+import com.github.manolo8.darkbot.gui.utils.Popups;
+
+import javax.swing.*;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class StartupParams {
+    private static final String COMMAND_PREFIX = "-";
+
+    /**
+     * <b>NAME</b>
+     * <pre>
+     *     login - auto logins to the account with this <i>username</i>
+     * </pre>
+     * <b>SYNTAX</b>
+     * <pre>
+     *     -login <i>username</i> <i>filepath</i>
+     * </pre>
+     * <b>OPTIONS</b>
+     * <pre>
+     *     <i>username</i> and <i>filepath</i> are required arguments
+     *     <i>username</i> refers to the account you want to login to
+     *     <i>filepath</i> refers to the filepath to your txt file containing your master password
+     * </pre>
+     * <b>SAMPLE USAGE</b>
+     * <pre>
+     *     -login NotABot C:\Users\Owner\masterpw.txt
+     * </pre>
+     * @see #masterPasswordFile
+     */
+    private static final String LOGIN_COMMAND = COMMAND_PREFIX + "login";
+    private boolean autoLogin = false;
+    private String username;
+    /**
+     * This is a txt file containing your master file<br>
+     * Sample content of master password file:
+     * <pre>
+     *     password="mypassword12345"
+     * </pre>
+     * Leave it empty if you didnt set a master password like this:
+     * <pre>
+     *     password=""
+     * </pre>
+     */
```

Comment:
```suggestion
    /** Plaintext file containing the master password */
```
I'm not sure that verbose comments really help much